source/administration/changelog.md

```diff
@@ -2,6 +2,224 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.2.0
+
+Expected release date: 2016-07-16
+
+### Highlights
+
+#### Custom Emoji
+
+- Create Custom Emoji from the **Main Menu** > **Custom Emoji** when enabled from the **System Console** > **Customization** > **Custom Emoji**.
+- Restrict the permissions required to create Custom Emoji (Enterprise).
+
+#### SAML Single-Sign-On (Enterprise):
+
+- New Mattermost user accounts are automatically created the first time a user signs in with their SAML credentials. Mattermost pulls user information from SAML, including first and last name, email and username.
```

Comment:
Propose adding `Users can sign in to Mattermost with their SAML credentials`, e.g.:

`Users can sign in to Mattermost with their SAML credentials and new Mattermost user accounts are automatically created on first login. Mattermost pulls user information from SAML, including first and last name, email and username.`